pillar/heroku/xpro.sls

```diff
@@ -1,34 +1,54 @@
 {% set app_name = 'xpro' %}
-{% set environments = ['ci', 'rc', 'production'] %}
-{% set status_token = salt.random.get_str(24) %}
+{% set status_token = __vault__:gen_if_missing:secret-operations/global/xpro/django-status-token>data>value %}
+{% set env_dict = {
+    'ci': {
+      'env_name': 'ci',
+      'ga_id': '',
+      'release_branch': 'master',
+      'log_level': 'DEBUG'
+      },
+    'rc': {
+      'env_name': 'rc',
+      'ga_id': '',
+      'release_branch': 'release-candidate',
+      'log_level': 'INFO'
+      },
+    'production': {
+      'env_name': 'production',
+      'ga_id': '',
+      'release_branch': 'release',
+      'log_level': 'WARN'
+      }
+} %}
 
 proxy:
   proxytype: heroku
 
 heroku:
-  {% for env in environments %}
-  app_name: {{ app_name }}-{{ env }}
+  {% for env_data in env_dict %}
```

Comment:
should this be an iteration over the environment, or should it pull an environment grain from the proxy minion to populate the appropriate values?